tahoe_sites/models.py

```diff
@@ -0,0 +1,62 @@
+import uuid
+
+from django.conf import settings
+from django.db import models
+from organizations.models import Organization
+
+
+class UserOrganizationMapping(models.Model):
+    """
+    User membership in an organization.
+
+    Tahoe's fundamental multi-site relationship.
+    """
+
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+    is_active = models.BooleanField(default=True)
+    is_amc_admin = models.BooleanField(default=False)
+
+    class Meta:
+        app_label = 'tahoe_sites'
+
+    def __str__(self):
+        org = self.organization
+        return 'UserOrganizationMapping<{email}, {organization}>'.format(
+            email=self.user.email,  # pylint: disable=no-member
+            organization=org.short_name,
+        )
+
+
+class TahoeSiteUUID(models.Model):
+    """
+    Handle Site UUID
+    """
+    organization = models.OneToOneField(
+        Organization,
+        on_delete=models.CASCADE,
+        null=False,
+    )
+    site_uuid = models.UUIDField(default=uuid.uuid4, unique=True)
+
+    class Meta:
+        app_label = 'tahoe_sites'
+
+    def get_tier_for_org(self):
```

Comment:
This helper should live in the edx-platform until we have a better place for it:

We have the `get_active_sites()` helper in this `openedx/core/djangoapps/appsembler/sites/utils.py` file, I suggest moving this helper there and make it a standalone helper without a class:

```py
# appsembler/sites/utils.py
def get_tier_for_org(org):
   ...
```

Reply:
done